### ISSUE TITLE
Java api - unscaledValue() and scale()

### ISSUE THREAD

**Issue by zorba128** [epam/DFP#33]
I miss `Decimal64.unscaledValue(): Long` and `Decimal64.scale(): Int` that would allow to unpack value into mantissa/exponent in a way similar to `BigDecimal.scale()` and `BigDecimal.unscaledValue()`.

It would allow conversions like 
```
   val v1 = Decimal64.fromDecimalDouble(123.4567)
   val v2 = Decimal64.fromFixedPoint(v1.unscaledValue(), v1.scale()) //1234567L, 4
   assert(v1.isEqual(v2))
```

For now I go through BigDecimal, but this is far from optimal and unnecessary.

Such operation would be very useful for converting to other representations, or sending data over wire in more human-friendly way.
Actually Decimal64Parts might be used (public), but I found no way to create it in user code (JavaImpl is private, unlike JavaImplDiv/JavaImplMul).

**Comment by agdavydov81** [comment]
OK, I'll create and interface for decimal unpack and pack.

**Comment by agdavydov81** [comment]
Pull request https://github.com/epam/DFP/pull/36

**Comment by agdavydov81** [comment]
Fix published in 0.11.26.
